### DISCUSSION TITLE
Add visualization for countries with load data or "unknown" production only

### DISCUSSION THREAD

**Discussion by alixunderplatz** [electricitymaps/electricitymaps-contrib#2494] [General]
- There are quite a few new countries now which have a parser for "system load" aka "fetch_consumption()", now the "Gulf Cooperation Council" parser was added in #2360. 
Vietnam is another candidate for getting a load parser soon (see #1974). Even for Belarus it is possible to parse the system load (#934). Load data for all states of India is also available in real-time (#304) .

- Some more real-time "unknown production" could possibly be added for some zones. This is the case for the Mexican regions, since that data is given on the source website (see #1242). 

- There are more zones with a lack of production by type, so "total production" is going into "unknown". I think of Japan, where all these zones remain grey on the map, although there is some data (now there is even solar for all JP-zones, so this will probably change soon resolving #2448). 

Combining all these things above, I have two remarks:
1. Is it possible to indicate or color countries with "generic" production or consumption data? (different zone outlines, filling style, ...) 
At the moment, you wouldn't guess there is visible, useful data for Japan unless you select the zones.

2. Is it possible to have another view for the "fetch_consumption()" data, or is this for your electricityMap's internal purposes only (like forecasting, services, etc)?

I am aware it has little to do with showcasing the real-time carbon emissions of the zones and that implementing will cost time and effort. As I said in #2362, I just really value having load or production profiles of many different zones visualized, because they vary based on climate, wealth, local resources, season, ... And most responsible entities just totally "fail" to visualize and disclose such data properly, which leads to (accepted?!) lack of transparency.
For many researchers / analysts / students / politicians / citizens... this could contribute to (re-)evaluating renewable potentials, required storage sizes, load shifting, political views etc etc...

Data transparency and data visualization are the keys to fix this world in regard to its environmental challenges - and both are enforced by the Tomorrow-team and community. :) 🗺️ 

So, long story short: how are the chances for having a third toggle option feature
"production / consumption **/ load**" 
and/or 
a new color (like a different grey) for zones with data? 😃 (The latter could even fix zones with temporary data issues.^^)

**Comment by alixunderplatz** [comment]
Blue zones have available real-time system load data in addition to the currently featured zones:

![grafik](https://user-images.githubusercontent.com/25743609/81512172-de885500-931e-11ea-9250-9dcbef3afd5e.png)

**Comment by fbarl** [comment]
Thanks for a very detailed write up @alixunderplatz! :+1: 

I don't feel qualified to say which approach would be more useful (adding a _load_ option to the toggle vs. showing it in the current maps), but it generally sounds like a good idea to visualize these zones _somehow_.

A tiny UX remark regarding your screenshot above - I'd perhaps use a subtler color for such zones than bright blue as those zones should ideally get more visual attention than the gray zones but still less than the zones with full data - but to address that, we could e.g. simply use a "grayer" shade of blue or show it striped :slightly_smiling_face:

**Comment by systemcatch** [comment]
We could use a gray striped fill for countries with load data, adding a separate tab seems to up the complexity quite a bit.

**Comment by q--** [comment]
>There are more zones with a lack of production by type, so "total production" is going into "unknown". I think of Japan, where all these zones remain grey on the map, although there is some data (now there is even solar for all JP-zones, so this will probably change soon resolving #2448).


The gray Japanese zones also say "Data temporarily unavailable" when I mouse-over them, should I file an issue for that?
